### PR TITLE
Clean up unimplemented features and show legacy actions

### PR DESCRIPTION
**Pull request by alex-deaconu**
## Description
Clean up unimplemented features and show legacy actions

Hide credit cards and show edit payment methods button
Hide subscription actions and show edit subscription
Hide yearly and early invoice features
Hide tax exemption button and show no tax exemption message

[stage-19]

## Motivation and Context
Billing Frictions epic. Incremental release.

## How Has This Been Tested?
Tested changes locally. Updated unit tests.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc @santiagonoguez as discussed, hiding some elements and adding actions to allow for an interim release to production. Please review. Thanks!

**Comment by alex-deaconu** [comment]
Updated test coverage as well in the last commit. Thanks.

**Review by santiagonoguez — COMMENTED** [review]
@alex-deaconu looks good to me, and works ok here ( shows old functionality ), although there are some parts that I'm not completely familiar on the previous code.

File: web/scripts/billing/controllers/ctr-subscription.js [hunk 1]
```diff
@@ -22,5 +23,26 @@ angular.module('risevision.apps.billing.controllers')
         return subscriptionFactory.item && !subscriptionFactory.item.card;
       };
 
+      var _reloadSubscription = function () {
+        $loading.start('subscription-loader');
```
Comment (ezequielc) [inline web/scripts/billing/controllers/ctr-subscription.js:27]: @alex-deaconu When I updated a plan, the spinner didn't show. The subscription was reloaded as expected, just the spinner didn't show. Does it work for you? Thanks

File: web/scripts/billing/controllers/ctr-billing.js [hunk 2]
```diff
@@ -34,30 +32,10 @@ angular.module('risevision.apps.billing.controllers')
...
-        }, 10000);
-      };
-
-      $rootScope.$on('chargebee.subscriptionChanged', _reloadSubscriptions);
-      $rootScope.$on('chargebee.subscriptionCancelled', _reloadSubscriptions);
       $rootScope.$on('risevision.company.planStarted', function () {
         $scope.subscriptions.doSearch();
       });
 
-      $scope.editPaymentMethods = function () {
```
Comment (ezequielc) [inline web/scripts/billing/controllers/ctr-billing.js:51]: @alex-deaconu We need to either remove the Edit Payment section from the main billing page, or keep this method while it is there. The link is now broken. Thanks

File: web/scripts/billing/controllers/ctr-subscription.js [hunk 3]
```diff
@@ -22,5 +23,26 @@ angular.module('risevision.apps.billing.controllers')
...
+
+        $timeout(function () {
+          subscriptionFactory.reloadSubscription();
+        }, 10000);
```
Comment (ezequielc) [inline web/scripts/billing/controllers/ctr-subscription.js:31]: @alex-deaconu Given we are calling Chargebee directly (i.e. no webhook), I think we can remove the 10s delay here. Thanks

File: web/partials/billing/subscription.html [hunk 4]
```diff
@@ -24,9 +28,13 @@ <h4 class="mb-4">
           <label>Display Licenses</label>
           <h1>11</h1>
```
Comment (ezequielc) [inline web/partials/billing/subscription.html:29]: @alex-deaconu Just a heads up, `11` here is still hard coded.
Also, for other types os subscriptions, I am not sure what we should show here. Maybe just repeat the plan name? Thanks

**Comment by alex-deaconu** [comment]
@santiagonoguez thanks, @ezequielc wondering if you could take a quick look? Thanks.

File: web/scripts/billing/controllers/ctr-billing.js (hunk 2 above)
Comment (alex-deaconu) [reply web/scripts/billing/controllers/ctr-billing.js:51]: @ezequielc good catch, missed removing from the partial. Thanks.

File: web/scripts/billing/controllers/ctr-subscription.js (hunk 1 above)
Comment (alex-deaconu) [reply web/scripts/billing/controllers/ctr-subscription.js:27]: @ezequielc interesting, probably related to the reset, I will take a look. Thanks.

File: web/scripts/billing/controllers/ctr-subscription.js (hunk 3 above)
Comment (alex-deaconu) [reply web/scripts/billing/controllers/ctr-subscription.js:31]: @ezequielc good point, will update, thanks!

File: web/partials/billing/subscription.html (hunk 4 above)
Comment (alex-deaconu) [reply web/partials/billing/subscription.html:29]: @ezequielc ouch! Will fix, good catch. Thanks.

Comment (alex-deaconu) [reply web/partials/billing/subscription.html]: @ezequielc this is now fixed. I'm showing the count however I will work on showing something else for other subscriptions in a subsequent PR. Thanks.

File: web/scripts/billing/controllers/ctr-subscription.js (hunk 3 above)
Comment (alex-deaconu) [reply web/scripts/billing/controllers/ctr-subscription.js]: @ezequielc fixed. Thanks.

File: web/scripts/billing/controllers/ctr-billing.js (hunk 2 above)
Comment (alex-deaconu) [reply web/scripts/billing/controllers/ctr-billing.js:51]: @ezequielc removed. Thanks.

File: web/scripts/billing/controllers/ctr-subscription.js (hunk 1 above)
Comment (alex-deaconu) [reply web/scripts/billing/controllers/ctr-subscription.js]: @ezequielc this was removed when removing the timeout. Thanks.

**Comment by alex-deaconu** [comment]
@ezequielc addressed your review points minus what to show for non Display License plans. I'd like to work on that in a separate PR. Thanks.

**Review by ezequielc — APPROVED** [review]
@alex-deaconu Looks great 👍 Thanks for the updates

**Pull request merged**